.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "node_modules"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
I don't think these are correct settings - we're not using Python. Also, your individual VS Code settings should not be committed. They should be added to the `.gitignore` if needed.